hack/create_bmcs.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# This script reads BMC information in a config file and prepare VMs 
+# whose info match those config
+#
+set -eux
+
+REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/..")
+cd "${REPO_ROOT}" || exit 1
```

Comment:
```suggestion
cd "${REPO_ROOT}"
```
Unnecessary as we have `set -e` which will exit on error.